modules/test/playwright/tests/export-import-web/pages/CompanyExportImportPage.ts

```diff
@@ -39,6 +44,40 @@ export class CompanyExportImportPage {
 			await this.exportImportPage.exportPermissionsButton.click();
 		}
 
+		if (endDate && endTime && startDate && startTime) {
+			await this.exportImportPage.rangeDateRangeRadioButton.check();
+
+			await this.page
+				.locator(
+					'[id="_com_liferay_exportimport_web_portlet_CompanyExportPortlet_endDate"]'
```

Comment:
Maybe we could declare this `endDate`, `endTime`, `startDate`, `startTime` and `rangeLast` locators into the constructor so we have them just in case we want to use them later, and to provide readability to the test.